### PR TITLE
[sql lab] handle large ints, prevent overflow

### PR DESCRIPTION
**Pull request by mistercrunch**

### CODE REVIEW

File: superset/assets/spec/javascripts/sqllab/actions_spec.js [hunk 1]
```diff
@@ -86,13 +86,6 @@ describe('async actions', () => {
       expect(dispatch.args[0][0].type).to.equal(actions.START_QUERY);
     });
 
-    it('calls querySuccess on ajax success', () => {
```
Comment (mistercrunch) [inline superset/assets/spec/javascripts/sqllab/actions_spec.js]: Was a dupe to test on line 58

**Comment by betodealmeida** [comment]
Nice!

**Comment by williaster** [comment]
@mistercrunch this has to be adde back to `@superset-ui/core` now. can you please provide a description of the problem so I can port it / know how to test it properly?

**Comment by williaster** [comment]
I think the easiest way to support this is by exposing `dataType` in `SupersetClient`

**Comment by mistercrunch** [comment]
@williaster the issue is around JSON parsing when JSON contains big integer. JS does not support large ints and just overflows (number parsed out is different to the number in the JSON).

**Comment by mistercrunch** [comment]
screen shot 2018-09-12 at 5 29 41 pm

**Pull request merged**